modules/installing-ocp-agent.adoc

```diff
@@ -0,0 +1,167 @@
+----
+
+. Place the `openshift-install` binary in a directory that is on your PATH.
+
+. Create a directory to store the cluster manifests.
++
+[source,terminal]
+----
+$ mkdir ~/<directory_name>
+----
+
+[NOTE]
+====
+This is the preferred method for the Agent-based installation. Using ZTP manifests is optional.
+====
+
+. Create the `install-config.yaml` file.
++
+[source,yaml]
+----
+cat << EOF > ./cluster-manifests/install-config.yaml
+apiVersion: v1
+baseDomain: test.example.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  replicas: 0
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  replicas: 1
+metadata:
+  name: sno-cluster <1>
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 192.168.111.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  none: {}
+pullSecret: '<pull_secret>' <2>
+sshKey: |
+  '<ssh_pub_key>' <3>
+  EOF
+----
++
+<1> Required.
+<2> Enter your pull secret.
+<3> Enter your ssh public key.
+
+. Create the `agent-config.yaml` file.
++
+[source,yaml]
+----
+  cat > agent-config.yaml << EOF
+  kind: AgentConfig
+  metadata:
+    name: sno-cluster
+  rendezvousIP: 192.168.111.80 <1>
+  hosts: <2>
+    - hostname: master-0
+      interfaces:
+        - name: eno1
+          macAddress: 00:ef:44:21:e6:a5
+      networkConfig:
+        interfaces:
+          - name: eno1
+            type: ethernet
+            state: up
+            mac-address: 00:ef:44:21:e6:a5
+            ipv4:
+              enabled: true
+              address:
+                - ip: 192.168.111.80
+                  prefix-length: 23
+              dhcp: false
+        dns-resolver:
+          config:
+            server:
+              - 192.168.111.1
+        routes:
+          config:
+            - destination: 0.0.0.0/0
+              next-hop-address: 192.168.111.2
+              next-hop-interface: eno1
+              table-id: 254
+  EOF
+----
++
+<1> This IP address is used to determine which node performs the bootstrapping process as well as running the `assisted-service` component.
+You must provide the IP address when you do not specify the node's IP addresses in `networkConfig`. If this address is not provided, one IP address is selected from the provided nodes's `networkConfig`.
+<2> The number of hosts defined must match the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters. When 3 master nodes and 0 worker nodes are defined in the `install-config.yaml` file,
+the number of hosts defined is 3. When 3 master nodes and 2 worker nodes are defined in the `install-config.yaml` file, the number of hosts defined is 5.
+
++
+. Create the Agent image.
++
+[NOTE]
+====
+Ensure that the image is created along with the **auth** folder. You can verify this with the following command:
```

Comment:
Does this NOTE take place before or after running $ openshift-install agent create image
? We could make it an optional step so that it's not a note, like follows:

. (Optional) Ensure that the image is created with the `auth` folder. To verify this, enter the following command: 
+
[source,terminal]
----
$ ls
agent.iso  auth
----

Reply:
I would like to retain the note. I would however re-phrase the sentence.